### PR TITLE
Offline transformations exposed to python with pybind11

### PR DESCRIPTION
**Pull request by tomdol**
### Tickets:
 - 61868

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
can we completely remove offline transformations from old API?
It's internal and not needed for BW compatibility

**Comment by tomdol** [comment]
I've used a new `get_function` binding in this PR but it will only be available after https://github.com/openvinotoolkit/openvino/pull/8027 has been merged. Some CI failures are probably caused by this.

**Review by slyubimt — APPROVED** [review]

**Review by akuporos — APPROVED** [review]

File: runtime/bindings/python/src/pyopenvino/CMakeLists.txt [hunk 1]
```diff
@@ -51,12 +57,12 @@ if(OpenVINO_SOURCE_DIR OR InferenceEngineDeveloperPackage_FOUND)
...
 
     install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/../openvino
-            DESTINATION python/${PYTHON_VERSION}
+            DESTINATION ./python/${PYTHON_VERSION}
             COMPONENT pyopenvino_${PYTHON_VERSION}
             USE_SOURCE_PERMISSIONS)
 
     install(TARGETS ${PROJECT_NAME}
-            DESTINATION python/${PYTHON_VERSION}/openvino
+            DESTINATION ./python/${PYTHON_VERSION}/openvino
```
Comment (slyubimt) [inline runtime/bindings/python/src/pyopenvino/CMakeLists.txt:65]:
```suggestion
            DESTINATION python/${PYTHON_VERSION}/openvino
```

File: runtime/bindings/python/src/pyopenvino/CMakeLists.txt [hunk 2]
```diff
@@ -51,12 +57,12 @@ if(OpenVINO_SOURCE_DIR OR InferenceEngineDeveloperPackage_FOUND)
     ie_cpack_add_component(pyopenvino_${PYTHON_VERSION})
 
     install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/../openvino
-            DESTINATION python/${PYTHON_VERSION}
+            DESTINATION ./python/${PYTHON_VERSION}
```
Comment (slyubimt) [inline runtime/bindings/python/src/pyopenvino/CMakeLists.txt:60]:
```suggestion
            DESTINATION python/${PYTHON_VERSION}
```

**Review by slyubimt — CHANGES_REQUESTED** [review]

**Review by slyubimt — APPROVED** [review]

**Pull request merged**